<launch>
  <node name="pioneer" pkg="p2os" type="p2os">
    <remap from="gripper_control" to="/gripper_command"/>
    <remap from="cmd_vel" to="/velocity_command"/>
    <remap from="cmd_motor_state" to="/motor_state_command"/>
    <remap from="ptz_control" to="/ptz_command"/>
  </node>

  <node name="pioneer_control" pkg="pioneer_control" type="pioneer_control_node.py">
    <remap from="gripper_status" to="/gripper_state"/>
    <remap from="motor_status" to="/motor_state"/>
    <remap from="ptz_status" to="/ptz_state"/>
  </node>
</launch>



